modules/nodes-descheduler-installing.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-installing_{context}"]
+= Installing the descheduler
+
+// TODO:
+The descheduler is not available by default. To enable the descheduler, you must install the Descheduler Operator from OperatorHub. After the Descheduler Operator is installed, you can then configure the eviction strategies.
+
+.Prerequisites
+
+// TODO: must be cluster admin?
```

Comment:
For installing you'll need cluster admin to be able to create the proper roles and service accounts